spec/controllers/assignment_invitations_controller_spec.rb

```diff
@@ -0,0 +1,86 @@
+require 'rails_helper'
+
+RSpec.describe AssignmentInvitationsController, type: :controller do
+  describe 'GET #show' do
+    let(:invitation) { create(:assignment_invitation_with_assignment) }
+
+    describe 'unauthenticated request' do
+      it 'will redirect the new user to sign in with GitHub' do
```

Comment:
Minor nit: `it 'redirects...'`

This is a lot like writing a commit message, drop any unnecessary words to keep it clear.